dashboards/secops/secops.json

```diff
@@ -0,0 +1,407 @@
+{
+  "displayName": "SecOps Exporter",
+  "dashboardFilters": [],
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 15,
+        "width": 48,
+        "widget": {
+          "title": "SecOps Request Latency P99, P95, P50",
+          "id": "",
```

Comment:
This can be removed. Same for other widgets as well on line 110, 178, 260, 305 and 349.